lab2/configmap/nginx-config.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nginx-config
```

Comment:
it makes more sense to have the DocRoot dir in the pv and the nginx conf file in your configmap

Reply:
Yes, but the lab instructions contain :
5. create cm containing index.html file content the content is <center> Hello Hossam </center>